autoload/go/complete.vim

```diff
@@ -16,21 +16,12 @@ function! s:gocodeCommand(cmd, args) abort
 endfunction
 
 function! s:sync_gocode(cmd, args, input) abort
-  " We might hit cache problems, as gocode doesn't handle different GOPATHs
-  " well. See: https://github.com/nsf/gocode/issues/239
```

Comment:
Is this no longer applicable? Last time this was touched it was mentioned that this is needed: https://github.com/fatih/vim-go/pull/1525#discussion_r145573584